### DISCUSSION TITLE
We should add more ways to brain wash into the game. *GONE WRONG*

### DISCUSSION THREAD

**Discussion by ShikhaliyevRustam** [ParadiseSS13/Paradise#20681] [Documents]
There should be an ability to put brains washing machines to rid those pesky dirty thoughts in our minds. Did you know that washing your brain is linked with curing all sorts of mental ailments, and loved ones will compliment you on your clean mind.

**Comment by S34NW** [comment]
https://user-images.githubusercontent.com/12197162/226430072-5fc33709-97f0-4ff8-8d59-5be90b21e32e.mp4

**Reply by ShikhaliyevRustam** [reply]
I am always happy whenever there is a red alert 2 reference!
